rerun_py/rerun_sdk/rerun/notebook.py

```diff
@@ -0,0 +1,77 @@
+"""Helper functions for displaying Rerun in a Jupyter notebook."""
+
+import base64
+import logging
+import random
+import string
+from typing import Any, Optional
+
+from rerun import bindings
+
+
+def inline_show(width: int = 950, height: int = 712, app_location: Optional[str] = None, timeout_ms: int = 2000) -> Any:
+    """
+    Show the Rerun viewer in a Jupyter notebook.
+
+    Parameters
+    ----------
+    width : int
+        The width of the viewer in pixels.
+    height : int
+        The height of the viewer in pixels.
+    app_location : str
+        The location of the Rerun web viewer.
```

Comment:
Location how? The web-server URL of the web-viewer `index.html`? Perhaps `web_viewer_url` is a better name?